CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Snyk Security Changelog
 
-
 ## [2.9.0]
```

Comment:
Are we planning on a manual minor release? The weekly schedule does patch releases.

Reply:
Thanks that's a good point! I didn't really know when the release does a minor vs patch. I'll change this and fix the previous CHANGELOG message too